src/components/LoginDropdown.js

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import { Menu, Transition } from "@headlessui/react";
+import { personCircleSharp } from "ionicons/icons";
+import { IonIcon } from "@ionic/react";
+import Button from "./Button";
+
+export default function Dropdown({ onClick = () => {} }) {
```

Comment:
I think we should either rename this component to `LoginDropdown`..., or rename the name of the file to `src/components/Dropdown.js` so that there's consistency.